src/odemis/gui/cont/microscope.py

```diff
@@ -138,6 +139,9 @@ def __init__(self, btn_ctrl, va, main_data):
         if model.hasVA(main_data.chamber, "pressure"):
             main_data.chamber.pressure.subscribe(self._on_pressure_change, init=True)
 
+        # TODO: support alternative pressure label (e.g. TextControl below button)?
```

Comment:
How do you show the pressure now, if not on the button?

Reply:
I don't show it, it's either pumped (green) or vented (black). Do you think it's a good idea to also show the numerical value?

Reply:
Ok. From the TODO I thought there is still somewhere the pressure shown, as it says "alternative pressure label". I don't think we show the pressure in the SPARC, but not sure. Easy check you can do with the simulator. Job would be the right person to say if we should display it or not. Maybe you can rephrase your TODO or delete it if Job says it is not needed.

Reply:
I updated the PR to show the pressure in a label below the buttons.